src/customize-pop/appearance-settings.md

```diff
@@ -0,0 +1,13 @@
+# Appearance Settings
+
+This section describes the Light and Dark theme options located in the `Appearance` tab. These themes apply to all windows across the operating system. Selecting a theme also assigns a default desktop background.
```

Comment:
> Selecting a theme also assigns a default desktop background.

This kind of makes it sound like the background will be reset to default when selecting a theme (this is how it works in Windows, or at least used to in Vista/7.) That is not the case. Each theme comes with its own default desktop background, but if the user has already assigned a custom background, then the background will not be modified.

Reply:
Clarified